nautobot_chatops/dispatchers/slack.py

```diff
@@ -36,6 +37,7 @@ def __init__(self, *args, **kwargs):
         """Init a SlackDispatcher."""
         super().__init__(*args, **kwargs)
         self.slack_client = WebClient(token=settings.PLUGINS_CONFIG["nautobot_chatops"]["slack_api_token"])
+        self.slack_menu_limit = 100
```

Comment:
Let's move this is to a configurable setting.

```suggestion
        self.slack_menu_limit = int(os.getenv("SLACK_MENU_LIMIT", 100))
```

Reply:
Updated